core/trading_engine.py

```
@@ -639,46 +713,58 @@ async def _collect_market_data(self) -> None:
             exchange_names.append(exchange_name)
-        
+
         # 모든 거래소 데이터 동시 수집
         if tasks:
+            logger.info(f"🔍 [MARKET-DATA] {len(tasks)}개 거래소 데이터 수집 시작")
             results = await asyncio.gather(*tasks, return_exceptions=True)
-            
+            logger.info(f"🔍 [MARKET-DATA] 데이터 수집 완료, 결과 처리 중")
+
             # 결과 처리
             for exchange_name, result in zip(exchange_names, results):
```

Ruff (0.14.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)